src/main/java/Duke.java

```diff
@@ -1,10 +1,171 @@
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class Duke {
+
+    public static final String LINE = "_______________________________________\n";
+    private static ArrayList<Task> tasks = new ArrayList<Task>();
+    private static File file = new File("C:\\Users\\weich\\Documents\\CS2113T\\CS2113T\\ip\\src\\main\\java\\duke.txt");
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+        String logo = " ____        _        \n" + "|  _ \\ _   _| | _____ \n" + "| | | | | | | |/ / _ \\\n" + "| |_| | |_| |   <  __/\n" + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        System.out.println("Hello! I'm Duke\n" + "What can I do for you?\n" + LINE);
+        inputTask();
     }
-}
+
+    private static void writeToFile() throws IOException {
+        FileWriter fw = new FileWriter(file);
+        for (int i = 0; i < tasks.size(); i++) {
+            fw.write(tasks.get(i).toString() + System.lineSeparator());
+        }
+        fw.close();
+    }
+
+    private static void saveToFile() throws IOException{
```

Comment:
Is there a bug in this method? I don't quite get what it's trying to do, it seems like it is not used in your code too (if it is not used, you would want to remove it).
Otherwise, you may want to comment on the difference between writeToFile and saveToFile, looks to be the same purpose to me.